packages/notifications/resources/views/components/close-button.blade.php

```diff
@@ -1,4 +1,4 @@
 <x-heroicon-s-x
-    class="filament-notifications-close-button h-6 w-5 cursor-pointer text-gray-400"
+    class="filament-notifications-close-button h-4 w-4 cursor-pointer text-gray-400"
```

Comment:
I believe 20px size matches Tailwind UI.

Reply:
But it looks bad in this case 😬

Reply:
Hmm, the same icon will also be added to modals in #3911. We should try to be consistent, and I 20px size looks totally fine actually.